apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -275,6 +270,14 @@ function main() {
 
     $scope.hocCategoryName = i18n.hoc_category_name;
 
+    // Angular does not offer a reliable way to wait for the template to load,
+    // so do it using a custom event here.
+    $scope.$on('section-page-rendered', () => {
```

Comment:
There will presumably be some cleanup possible here once we can pull out the angular table (i.e. this is no longer an experiment). I think we will still need the angular for routing though, as that's what knows how to handle the `#/sections` in our URL.